spec/index.bs

```diff
@@ -282,10 +282,12 @@ interface GPUShaderStageBit {
 enum GPUBindingType {
     "uniform-buffer",
     "dynamic-uniform-buffer",
-    "sampler",
-    "sampled-texture",
     "storage-buffer",
-    "dynamic-storage-buffer"
+    "dynamic-storage-buffer",
```

Comment:
Perhaps we could find a better single-word alternative to "readonly-storage"? For example, "read-buffer" and possibly the corresponding "read-texture" instead of "sampled-texture".

Reply:
This is the same problem as discussed in #256: uniform buffers are read-only but use a different HW path compared to readonly storage buffers.

Reply:
I'm not suggesting to unify "uniform" and "read-storage" usages, here I'm only talking about the name for the latter.

Reply:
So `uniform-buffer` and `read-buffer`? It seems more confusing imho because it isn't clear for experienced GPU developers what the difference is between the two. Maybe `read-storage-buffer`?

Reply:
I just feel like "read-storage-buffer" is both unnecessary long and inconsistent with the texture one ("sampled-texture"). Experienced GPU developers are likely to recognize "uniform" and be able to distinguish it from other usages.